contributors/devel/pull-requests.md

```diff
@@ -77,35 +96,255 @@ $ $RELNOTES -man # for details on how to use the tool
+
+Here are the requirements the submit-queue checks automatically before it will merge your PR:
+
+* Author has signed the CLA ("cla: yes" label added to PR)
+* No changes made since last `lgtm` label applied
+* k8s-bot reported that the GCE E2E build and test steps passed (Jenkins unit/integration, Jenkins e2e)
+* On call contributor has applied the `ok-to-merge` label manually, unless you are on the [whitelist](https://github.com/kubernetes/contrib/blob/master/mungegithub/whitelist.txt) ??: this link seems to be broken
+
+# Why was my PR closed?
+
+Pull requests that are purely support questions will be closed and redirected to [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes). We do this to consolidate questions into a single channel, improve efficiency in responding to requests, and make FAQs easier to find.
+
+Pull requests older than 2 weeks will be closed. Exceptions can be made for PRs that have active review comments, or that are awaiting other dependent PRs. Closed pull requests are easy to recreate, and little work is lost by closing a pull request that subsequently needs to be reopened. We want to limit the total number of PRs in flight to:
 * Maintain a clean project
 * Remove old PRs that would be difficult to rebase as the underlying code has changed over time
 * Encourage code velocity
 
-For pull requests that are in progress but not ready for review, prefix the PR title with "WIP" and
-track any remaining TODOs in a checklist in the pull request description.
+# Why is my PR not getting reviewed?
+
+If it's the last few weeks of a milestone, we need to reduce churn and stabilize.
+
+Or, it could be related to best practices. One common issue is that the PR is too big to review. Let's say you've touched 39 files and have 8657 insertions. When your would-be reviewers pull up the diffs, they run away - this PR is going to take 4 hours to review and they don't have 4 hours right now. They'll get to it later, just as soon as they have more free time (ha!).
+
+We've got a detailed rundown of best practices, including how to avoid too-lengthy PRs, in the next section.
+
+But, if you've already followed the best practices and you still aren't getting any PR love, here are some
+things you can do to move the process along:
+
+   * Make sure that your PR has an assigned reviewer (assignee in GitHub). If not, reply to the PR comment stream asking for a reviewer to be assigned.
+
+   * Ping the assignee (@username) on the PR comment stream, and ask for an estimate of when they can get to the review.
+
+   * Ping the assigned on [Slack](http://slack.kubernetes.io). Remember that a person's github username might not be the same as their Slack username. 
+
+   * Ping the assignee by email (many of us have publicly available email addresses, or are the same as our GitHub handle @google.com or @redhat.com).
```

Comment:
I think we should remove the "or are the same as our GitHub handle @google.com or @redhat.com" bit